src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -1164,10 +1158,7 @@ public void SarifTransformerTests_ToVersionOne_ResultLocations()
                   ""message"": ""Foxes, quick and brown"",
                   ""locations"": [
                     {
-                      ""region"": {
-                        ""offset"": 1345,
-                        ""length"": 22
-                      }
+                      ""region"": {}
```

Comment:
Likewise, you shouldn't be getting an empty region here. It should just have `byteOffset` and `byteLength`.

Reply:
Didn't notice this.